mod/profiles/set/self/research_group.rb

```diff
@@ -0,0 +1,19 @@
+def subvariants
+  {
+    created: [:submitted, :organized]
+  }
+end
+
+def created_query user_id, variant=nil
+  case variant
+  when :submitted
+    { created_by: user_id }
+  when :organized
+    { right_plus: [OrganizerID, { refer_to: user_id }] }
+  else
+    { or:
+        created_query(user_id, :submitted).merge(
+          created_query(user_id, :organized)
+        ) }
```

Comment:
Doesn't seem very heavy to me that search. Tab switching on staging is pretty fast.